### PR TITLE
ERT is easier to call

### LINKED ISSUE #14837: ERT is to hard to call
Currently ERT is far far to hard to call. Without admin intervention to call a ERT you need to

- raise to red which requires 2 heads (reasonable)

- a captain level ID on the communications console (not head of staff level it has to have captain access)

- the shuttle to NOT be already called

On paper this sounds resonable and just but in practice it is not possible to call a ERT without a admin helping due to

- If it actually got so bad that a ERT would be called there would be a low chance of 2 heads being alive (yes you can promote an assistant to help but that is also a mess to explain how to raise to red)

- If you needed a ERT and had a comms console already then you would have called the shuttle. While it makes sense to not allow ert while shuttle is coming in practice nearly every time i have seen a ERT attempted to be called without admins helping the shuttle would already be on its way. It may not seem big but those 10 minutes can mean life or death for the entire crew

- Captains level access. Not many people play captain and the ones who do are usually to dumb to know players can call ert without praying to admins and since to have all access and not be captain/lone head of staff you need to be antag/powernigger who rushed the spare roundstart.

Maybe reduce the access to requirement to call ERT down to bridge level  (head of staff access), allow the ID swiper (same one used for raising to red) to allow for a swipe for ERT, and allow ERT to be called when the shuttle is called because 99% of the time when a ERT is needed the shuttle is already en route.

Also think when was the last time you remember seeing a ERT get called WITHOUT admins helping call it. Deathsquads dont count

### PR DESCRIPTION
**Pull request by DamianX**
Follow-up to the discussion in #14837.
Closes #14837

:cl:
 * tweak: It is possible to call an ERT with a heads of staff ID, where previously a captain-access ID was needed. Also the ERT can be called once the shuttle is already en-route

### CODE REVIEW

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -192,13 +192,13 @@ var/shuttle_call/shuttle_calls[0]
...
-				to_chat(usr, "The emergency shuttle is already on its way.")
-				return
+			//if(emergency_shuttle.online)
+			//	to_chat(usr, "The emergency shuttle is already on its way.")
+			//	return
 			if(!(get_security_level() in list("red", "delta")))
 				to_chat(usr, "<span class='notice'>The station must be in an emergency to request a Response Team.</span>")
 				return
-			if(authenticated != 2 || issilicon(usr))
+			if(authenticated != 1 || issilicon(usr))
```
Comment (Kurfursten) [inline code/game/machinery/computer/communications.dm]: This means the captain can't use it. You could go for the more general if(!authenticated || issilicon(usr)) though - which is false for both values 1 and 2

**Comment by ultraplen** [comment]
>turns rare ebin-when-it-happens event into something that will happen every few rounds 

It may even reach the point where dead players will start expecting an ert so that they can get back into the round. Erts are going to be called in as soon as things start going south for the crew.

**Comment by DamianX** [comment]
...which could be okay? would you rather people call for ERT and deal with the situation or just call the shuttle and end the round right there?

**Comment by jknpj** [comment]
As long people don't start suiciding to join ERT it should be fine. Hell, people suicide to become xenomorph/wizard all the time, this should be ok.

**Comment by ultraplen** [comment]
My primary concern is that this will massively shift the balance in favour of the crew in team antag modes. ERTs will be called in as soon as someone shouts "cult maint" over the radio. The shuttle being called during team antag modes is the crew surrendering, which should happen a fair amount of the time.

Also, the ERTs are a lot of fun because they are semi-rare. Right now it's something really special to have the ERT roll in. Having them show up every second round would water them down significantly.

Having said all that, I do think this server is a bit trigger happy when it comes to calling the shuttle, but unleashing the e-gun valid squad as a go-to solution for preventing shuttle calls is a bad idea.

**Comment by Shadowmech88** [comment]
>ERTs will be called in as soon as someone shouts "cult maint" over the radio.

What prevents this from happening right now?

**Comment by ultraplen** [comment]
The whombo-combo of requiring captain access and not being able to call in the goons whilst the shuttle is en route. But even that is a bit thin in my opinion, it should be admin only. 

If this gets merged the CMO can summon them boiz to cover the crews retreat, assuming the station is on red. And why wouldn't he, when all he has to do is go press a button to resurrect half the crew and arm them with e-guns?

**Comment by Shadowmech88** [comment]
>The whombo-combo of requiring captain access and not being able to call in the goons whilst the shuttle is en route.

But the shuttle would not be en route at the time someone shouts "cult maint."
Nor, most likely, would the captain already be dead by that time.

So as long as there is a captain at all, I don't see what currently prevents him from calling an ERT as soon as a cult is spotted.

**Comment by ultraplen** [comment]
Fair, perhaps this PR won't be as game changing as I said it will. It does, however, allow for "non-sec" heads to have a huge impact on the round without the oversight of the acting captain. 

Also, being able to call in the stun-n-gun roundsaver brigade whilst the shuttle is coming will add nothing of value to the game. That tasty evac suspense will be replaced by a ttt death-match.

**Comment by Intigracy** [comment]
An alternative I can propose is adding ERT to the keycard authentication devices for the non-captain heads, so you can either have two heads call one or have the captain use the comms console.

**Comment by ghost** [comment]
You can call the ERT but it doesn't mean you'll get any ghosts to actually opt in. Which sums up the majority of my captain ERT calls. It's not going to be game changing ultraplen.

**Comment by Blithering** [comment]
@Intigracy ERT was already added to keycards, as was pointed out in the issue.

**Comment by ultraplen** [comment]
Gamechaning or not I think it's a step in the wrong direction for erts. Requiring 2 heads is a solid idea, though.

**Comment by Intigracy** [comment]
I believe the OP of the issue just wasn't aware of being able to call an ERT on code red from the keycard auth

**Comment by DamianX** [comment]
Should be good to merge.

**Comment by Intigracy** [comment]
Is this still an issue though now that more people know about the keycard authenticators?

**Comment by DamianX** [comment]
I dunno, I never cared much in the first place. Should I close this?

**Comment by SonixApache** [comment]
I still think this should be merged.

**Comment by Intigracy** [comment]
Shoulda did this a while ago, my bad.

http://ss13.moe/index.php/poll/118

If no wins, I'll change the keycard auth to not even need code red, instead it'll just bump to code red if used during non-red.

**Comment by Intigracy** [comment]
This will be revisited if ERT become too common.

**Pull request merged**